generator/uscore/uscore_generator.rb

```diff
@@ -642,6 +642,8 @@ def create_must_support_test(sequence)
             missing_must_support_elements = must_supports[:elements].reject do |element|
               #{resource_array}&.any? do |resource|
                 value_found = resolve_element_from_path(resource, element[:path]) { |value| element[:fixed_value].blank? || value == element[:fixed_value] }
```

Comment:
```suggestion
                value_found = resolve_element_from_path(resource, element[:path]) do |value|
                  value_without_extensions = value.to_hash.reject { |key, _| key == 'extension' } if value.respond_to? :to_hash
                  value_without_extensions.present? && (element[:fixed_value].blank? || value == element[:fixed_value])
                end
```

Reply:
`resolve_element_from_path` will only return elements that resolve to true when passed through the code block. So if the element is in an array, then this will look through the array for elements that are more than extensions.

Reply:
Moving this here would make it so that the new line 645 `value_found = value_found.to_hash.reject { |key, _| key == 'extension' } if value_found.respond_to? :to_hash` can be removed, right?  Because essentially we have filtered within this block, so we don't need to do it again after.

Reply:
I've addressed this in my most recent push. I had to manually add the changes since this code isn't quite right, and as you say, it requires changing multiple lines.